libr/config/config.c

```diff
@@ -217,8 +217,6 @@ R_API void r_config_list(RConfig *cfg, const char *str, int rad) {
 			if (!str || (str && (!strncmp (str, node->name, len)))) {
 				if (!str || !strncmp (str, node->name, len)) {
 					if (R_STR_ISNOTEMPTY (str)) {
-						cfg->cb_printf ("%s\n", r_str_get (node->desc));
```

Comment:
i think just adding a break after this will fix the issue. if you want to get the other output you can just use the internal grep. but imho the behaviour of `e?k` is to get only the description of `k`